packages/javascript/bh-shared-ui/src/components/HelpTexts/CodeController/CodeController.tsx

```diff
@@ -92,8 +92,8 @@ function CodeController(props: PropsWithChildren<Props>) {
         const perLine = (children?.toString() ?? '').split('\n');
         const nextNonBlankLine = perLine.find((x, i) => i !== 0 && !!x.trim());
 
-        const startingIndex = nextNonBlankLine?.split('').findIndex((x) => !!x.trim());
-        return perLine?.map((x) => x.slice(startingIndex)).join('\n');
+        const leftIndentationIndex = nextNonBlankLine?.split('').findIndex((x) => !!x.trim());
+        return perLine?.map((x, i) => i === 0 ? x : x.slice(leftIndentationIndex)).join('\n');
```

Comment:
This is a good catch. Would this still function as we intend if the second line has some indentation? Something like this for example:
\`First line
------Indented second line
  
Non indented third line\`

Reply:
```
first line
    indented second line
non indented third line
```

Reply:
^Yes, like that. Thank you for the formatting help @elikmiller!

Reply:
I believe in this case the 3 and subsequent lines would be trimmed to the second line indentation.... I think a way to handle this could be by removing only whitespace up to that left indentation index. 

I can update with that!

Reply:
This looks to provide enough guard rails and I appreciate the breakout from the inline map to a block of a few lines as it is easier to parse. There may be more edge cases with the formatting and the white space removal but since we want to move this content out of react components eventually this will hold us over.

Thanks for the update!

Reply:
Thank you and great call out! I dont love all the edge cases this has to handle, but I agree it should suffice for now until we replace it.